src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Introduction to GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Apr 24, 2024
+tags: CI, CD, github, githubAction
+banner: https://blog.jankaritech.com/src/assets/githubAction/images/githubaction_banner.png
+---
+GitHub Actions is an automation and CI/CD (Continuous Integration/Continuous Delivery or Deployment) service provided by GitHub, allowing us to automate our software development processes or workflows, from building and testing code to deploying it to different environments. It's an effective tool that can enhance software quality and save our time.
+
+## Some features of GitHub Actions:
+- It allows you to automate your Software Development Life Cycle (SDLC) processes or workflows.
```

Comment:
Does GitHub Actions automate the whole SDLC??

Reply:
NO, the statement `It allows you to automate your Software Development Life Cycle (SDLC) processes or workflows` does not imply automating the entire SDLC process. 
Instead, it suggests that GitHub Actions enables automation for certain aspects or stages of the SDLC. The term `processes or workflows` indicates that specific tasks or workflows within the SDLC can be automated using GitHub Actions, but it does not necessary cover the complete SDLC.

I updated that point, making it clear and understandable so that others would not be confused. 
Thank you :hugs: for your review.